### PR TITLE
models + webpage + visualizer

### PR DESCRIPTION
**Pull request by d1scrd**

### CODE REVIEW

File: Backend/Django/testApp/__pycache__/urls.cpython-312.pyc
Comment (d1scrd) [inline Backend/Django/testApp/__pycache__/urls.cpython-312.pyc:1]: Remove pyc files

Comment (d1scrd) [reply Backend/Django/testApp/__pycache__/urls.cpython-312.pyc:1]: delete everything in the pycache folder...

File: Desktop/src/components/ModelsCard.svelte [hunk 1]
```diff
@@ -33,174 +53,68 @@
...
-  //     try {
-  //     await loadScript('/home/kea_mothapo/Desktop/ACV_Project/Autonomous-Car-Vision/Desktop/src/routes/modelsJS/imagesloaded.pkgd.min.js');
-  //     await loadScript('/home/kea_mothapo/Desktop/ACV_Project/Autonomous-Car-Vision/Desktop/src/routes/modelsJS/charming.min.js');
-  //     await loadScript('/home/kea_mothapo/Desktop/ACV_Project/Autonomous-Car-Vision/Desktop/src/routes/modelsJS/TweenMax.min.js');
-  //     await loadScript('/home/kea_mothapo/Desktop/ACV_Project/Autonomous-Car-Vision/Desktop/src/routes/modelsJS/demo.js');
-  //     console.log('All scripts loaded successfully');
-  //     } catch (error) {
-  //     console.error(error);
-  //     }
-  // })
```
Comment (d1scrd) [inline Desktop/src/components/ModelsCard.svelte:58]: remove commented out code

File: Desktop/src/components/ModelsCardContent.svelte [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <script>
     import { Button, Tooltip } from "svelte-materialify";
+    // import { selectedModel } from "../stores/modelsStore";
     // Exported Parameters
```
Comment (d1scrd) [inline Desktop/src/components/ModelsCardContent.svelte]: remove comments

File: Desktop/src/components/ModelsCardContent.svelte [hunk 3]
```diff
@@ -11,24 +15,43 @@
       mProfileImg: "https://cdn.pixabay.com/photo/2024/03/11/19/15/ai-generated-8627457_640.png",
       mImg: "",
     };
+    //  export let key;
```
Comment (d1scrd) [inline Desktop/src/components/ModelsCardContent.svelte]: remove comments

File: Desktop/src/routes/Models.svelte [hunk 4]
```diff
@@ -112,83 +56,45 @@
...
+
+        </div>
+
+        {#if selectedModel}
+        <!-- svelte-ignore a11y-click-events-have-key-events -->
+          <div class="model-content-overlay" on:click={closeModelContent}>
+              <ModelsCardContent {selectedModel} />
+          </div>
+        {/if}
+
```
Comment (d1scrd) [inline Desktop/src/routes/Models.svelte:74]: remove comments

**Review by d1scrd — COMMENTED** [review]
please fix

File: Backend/Django/testApp/__pycache__/urls.cpython-312.pyc
Comment (keamothapo) [reply Backend/Django/testApp/__pycache__/urls.cpython-312.pyc:1]: done

**Review by d1scrd — COMMENTED** [review]
Looks good

**Review by JordanTimberlake — APPROVED** [review]
Good

**Pull request merged**